provisioner.go

```diff
@@ -358,6 +363,26 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 						},
 					},
 				},
+				{
+					Name: "script",
+					VolumeSource: v1.VolumeSource{
+						ConfigMap: &v1.ConfigMapVolumeSource{
+							LocalObjectReference: v1.LocalObjectReference{
+								Name: "script",
+							},
+							Items: []v1.KeyToPath{
+								{
+									Key:  "add.sh",
```

Comment:
We can call the script `setup`, `teardown`.

And it doesn't even need to be `.sh` since the interpreter should be specified at the first line of the script?

Reply:
i have changed add.sh&del.sh to setup&teardown